_posts/2020-01-26-newpipe-0.18.2-released.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "NewPipe 0.18.2 released: Hotfix for the YouTube-Extractor"
+short: "NewPipe 0.18.1 released"
+date: 2020-01-26 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+8 days after the last release and the next one is already out: A hotifx for the Extractor! A change on YouTubes side made it necessary.
+
+## Hotfix
+
+A hotfix release regarding the Extractor usually means that all the older versions of NewsPipe are likely to fail to play YouTube videos - this release is no difference. If you see any Guru Mediation error in the app, please update to this version before you create an issue (and/or verify that you actually have `0.18.2+` installed via hitting the `About` section in the left menu).
+
+<!-- more -->
+
+YouTube changed how they handled json. The actual fix was implemented by [@TobiGr](https://github.com/TobiGr), with the help of [@theScrabi](https://github.com/theScrabi) and (arguably even more helping) [@amuc](https://github.com/amuc). It was based on [@dstftw](https://github.com/dstftw) implementation in [youtube-dl](https://github.com/ytdl-org/youtube-dl). A big thanks goes to everyone involved.
+
+
+## Where to get the latest version
+
+NewPipe notifies you about new versions, you can download them when you press the notification. An alternative is the [Github release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), it notifies you as well about an update for [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/) - please keep in mind that it can take F-Droid a while to update their repository.
```

Comment:
Can you really "press" notifications? You can "click on" them perhaps.

Reply:
its a mobile, I aint clicking there